resources/Dockerfile

```diff
@@ -131,3 +131,29 @@ RUN if [ "$TARGETARCH" = "amd64" ]; then \
     && unzip ethr_linux.zip \
     && cp ethr /usr/local/bin \
     && rm ethr_linux.zip; fi
+
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
+    apt update; apt install golang-go -y \
+    && git clone https://github.com/Microsoft/ethr.git \
+    && cd ethr \
+    && go get golang.org/x/net/icmp \
+    && go get golang.org/x/net/ipv4 \
+    && go get golang.org/x/net/ipv6 \
+    && go build \
+    && cp ethr /usr/local/bin \
+    && cp ethr /tmp \
+    && cd .. \
+    && rm -rf ethr; fi
+
+# install fio from source code on arm64
+RUN if [ "$TARGETARCH" = "arm64" ]; then \
```

Comment:
Is fio not available from the distribution?

Reply:
Just started the arm64 docker image and fio is not in it, but somehow fio is available in the guest rootfs...So I think maybe the version of the docker container base image is the reason?

Reply:
I think you should be able to `apt install fio`: https://packages.ubuntu.com/focal/fio

Reply:
Ahh yes indeed, just checked and fio can be installed from apt:

```
root@6d3d6690e5e8:/cloud-hypervisor# apt install fio
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  libgfapi0 libgfrpc0 libgfxdr0 libglusterfs0 libtirpc-common libtirpc3
Suggested packages:
  gnuplot gfio python-scipy
The following NEW packages will be installed:
  fio libgfapi0 libgfrpc0 libgfxdr0 libglusterfs0 libtirpc-common libtirpc3
0 upgraded, 7 newly installed, 0 to remove and 22 not upgraded.
Need to get 943 kB of archives.
After this operation, 4109 kB of additional disk space will be used.
Do you want to continue? [Y/n] ^C

```

Reply:
Hello @MrXinWang @rbradford -, the default fio in current disk image "focal-server-cloudimg-arm64-custom-20210929-0.qcow2" doesn't support iouring. So we should build it from source and scp it into the image when do the test or put the fio into the image before test which means create a new image. Which do you prefer?

Reply:
We should use the same version / feature set as the version used on x86_64. I don't think whether fio supports io_uring or not matters from a testing perspective.

Reply:
Hi @rbradford -, do you mean we can using other ioengine than io_uring in fio test?

Reply:
My mistake. I didn't realised that the performance-metrics test was using the io_uring engine.

Reply:
I tried to add another commit https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3838/commits/7c98e5eae8aa8d3b72970bdc6656c677e22d8859 to solve this issue.